kalite/settings/base.py

```diff
@@ -285,7 +284,7 @@
     },
     "assessment_items": {
         "ENGINE": "django.db.backends.sqlite3",
-        "NAME": KHAN_ASSESSMENT_ITEM_DATABASE_PATH,
+        "NAME": __assessment_items_database_path,
```

Comment:
It's an internal and non-configurable variable, rather than something we expose *as if* it was configurable.